app/helpers/recruitment_banner_helper.rb

```diff
@@ -1,7 +1,17 @@
 module RecruitmentBannerHelper
-  TOPICS = ["/browse/business", "/browse/tax"].freeze
+  TOPICS = ["/browse/business", "/browse/tax", "/browse/employing-people"].freeze
 
   def show_banner?(path)
-    path.starts_with?(TOPICS.first) || path.starts_with?(TOPICS.last)
+    TOPICS.each do |topic|
+      return true if path.starts_with?(topic)
+    end
+
+    false
```

Comment:
```suggestion
TOPICS.any? { |topic| path.starts_with?(topic) }
```
I think this logic could be captured in a slightly shorter manner using `.any?` if it's easily understandable for other dev.

Reply:
I thought exactly this in the middle of the night! Thanks Aga